app/models/concerns/alegre_v2.rb

```diff
@@ -164,6 +201,7 @@ def get_context(project_media, field=nil)
         has_custom_id: true
       }
       context[:field] = field if field && is_not_generic_field(field)
+      context[:temporary_media] = project_media.class == TemporaryProjectMedia
```

Comment:
Suggestion / non-blocker: You could rewrite as `pm.is_a?(TemporaryProjectMedia)`.